docs/tutorials/jokes.md

```diff
@@ -2994,7 +2994,13 @@ export async function createUserSession(
 
 <summary>app/routes/login.tsx</summary>
 
-```tsx filename=app/routes/login.tsx lines=[18] nocopy
+```tsx filename=app/routes/login.tsx lines=[4,18] nocopy
```

Comment:
18 isn't the good line number anymore, can you update it ?

Reply:
Oh my bad...